### ISSUE TITLE
MQTT retain flag should be True

### ISSUE THREAD

**Issue by jsb5151** [roopesh/ad-qolsys#22]
I've experienced a few situations where discovery is failing because the topics simply don't show up in Mosquitto after a reconnect of the MQTT client, and this seems to be due to the fact that most messages sent by ad-qolsys have the Retain flag set to false.   I discovered this by trying to "fix" the other issue I flagged (#21) by manually pushing a discovery config, but it would still show up as Unavailable in the HA GUI.

Most if not all other MQTT integrations use Retain=True, why is this one not doing that?  I see Retain=true in some lines of the code but not sure why it doesn't apply to all of them.

**Comment by jsb5151** [comment]
That's true - I just created an automation that sends an INFO request to the panel, which fixes the states.  Not the most elegant solution but it works...

**Comment by roopesh** [comment]
@jsb5151 I do use a Will that makes the sensor unavailable.  That is a more accurate status.
